blackbox/docs/configuration.txt

```diff
@@ -701,6 +701,18 @@ Collecting Stats
   expressive results but also in more occupied RAM. Setting it to
   ``0`` disables collecting job information.
 
+.. _stats.jobs_log_expiration:
+
+**stats.jobs_log_expiration**
+  | *Default:*  ``5s``
+  | *Runtime:*  ``yes``
+
+  This setting specifies the duration how long jobs kept in the ``sys.jobs_log``
+  table. It periodically clears entries that are finished and older than the
+  specified time interval. This setting overrides ``jobs_log_size`` and logs a
+  warning message if both settings are set explicitly. If the value is set to
+  ``0`` the setting gets disabled.
```

Comment:
`the setting gets disabled.` does this mean that no job information will be collected ? 
if so maybe it should be said explicitly